doc/source/citations.rst

```diff
@@ -0,0 +1,63 @@
+
+
+
+************************
+Citations and references
+************************
+
+
+This document contains citation information for all data, method, or any
+relevant information used to make cimr work.
+
+
+
+===============================
+Genome-wide association studies
+===============================
+
+
+
+
+
+=================================================
+expression quantitative trait loci (eQTL) mapping
+=================================================
+
+-----------------------------------------
+Genotype-Tissue Expression project (GTEx)
+-----------------------------------------
+
+
+* The article containing the latest main analysis results:
+
+  The GTEx Consortium (2017).
+  Genetic effects on gene expression across human tissues. Nature. 550: 204–213.
+  https://www.nature.com/articles/nature24277.
+
+* The full summary-level data are on `GTEx Portal <https://gtexportal.org>`_
+
+* The raw and individualized data are on 
+  dbGaP `phs000424.v7.p2 <https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v7.p2>`_ 
+
+
+
+
+======================
+3D genome organization
+======================
+
+---------------
+TAD coordinates
+---------------
+
+* The article containing the method used to call TAD boundaries: 
+
+  Wang, Y., Song, F., Zhang, B., Zhang, L., Xu, J., Kuang, D., Li, D., 
+  Choudhary, M.N.K., Li, Y., Hu, M., et al. (2018). 
+  The 3D Genome Browser: a web-based browser for visualizing 3D genome 
+  organization and long-range chromatin interactions. Genome Biology. 19, 151. 
+  https://doi.org/10.1186/s13059-018-1519-9.
+
+* ENCODE TAD files are `here. <http://promoter.bx.psu.edu/hi-c/downloads/>`_ 
```

Comment:
I guess this is the correct syntax to have a link in RST file.

Reply:
it is. I hurriedly swapped between md and rst because markdown support for the readthedocs website is not good and forgot to change these links. Thanks for catching them!